config/dynamic.json

```diff
@@ -0,0 +1,138 @@
+{
+	"Dynamic": {},
+	"Roundstart": {
+		"Traitors": {
+			"cost": 8,
+			"scaling_cost": 9,
+			"weight": 0,
+			"required_candidates": 1,
+			"minimum_required_age": 0,
+			"requirements": [
+				10,
+				10,
+				10,
+				10,
+				10,
+				10,
+				10,
+				10,
+				10,
+				10
+			],
+			"antag_cap": {
+				"denominator": 24
+			},
+			"protected_roles": [
+				"Prisoner",
+				"Security Officer",
+				"Warden",
+				"Detective",
+				"Head of Security",
+				"Captain"
+			],
+			"restricted_roles": [
+				"Cyborg"
+			]
+		},
+
+		"Blood Brothers": {
+			"weight": 0
+		},
+
+		"Changelings": {
+			"weight": 0
+		},
+
+		"Heretics": {
+			"weight": 0
+		},
+
+		"Wizard": {
+			"weight": 0
+		},
+
+		"Blood Cult": {
+			"weight": 0
+		},
+
+		"Nuclear Emergency": {
+			"weight": 0
+		},
+
+		"Revolution": {
+			"weight": 0
+		},
+
+		"Families": {
```

Comment:
we don't have families??????????

Reply:
We don't.

I'll add it in at some point but I need to see if dynamic works.